### PR TITLE
created html file for results page -sv

### PR DESCRIPTION
**Pull request by sarahvaci**

### CODE REVIEW

File: results.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css"
+    />
+
+    <link rel="stylesheet" href="assets/css/style.css" />
+    <title> The Book Club Results</title>
+</head>
+
+<body>
+
+    <body>
```
Comment (ajsherrill2) [inline results.html]: I dont think we need this here as there should only be one body in the html

File: results.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
```
Comment (ajsherrill2) [inline results.html]: Can you put the entire nav section inside a header section

File: results.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+              <a href="#" class="brand-logo">The Book Club</a>
+              <ul id="nav-mobile" class="right hide-on-med-and-down">
+                <li><a href="homepage.html">Home</a></li>
+                <li><a href="#"></a></li>
+                <li><a href="#"></a></li>
+              </ul>
+            </div>
+          </nav>
+
+
```
Comment (ajsherrill2) [inline results.html]: can you put this div inside a main section

File: results.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                <li><a href="#"></a></li>
+              </ul>
+            </div>
+          </nav>
+
+        
+           <div class="title">
+               <h3> We Have Your Book Here! </h3>                                 
+                            
+                </div>
```
Comment (ajsherrill2) [inline results.html]: make sure there are no blank lines within single divs and that the opening and closing divs are both the same tab distance away from eachother

**Review by ajsherrill2 — CHANGES_REQUESTED** [review]

**Comment by sarahvaci** [comment]
Changes are updated.

**Review by ajsherrill2 — APPROVED** [review]

**Pull request merged**